### ISSUE TITLE
data downloads from landing page

### ISSUE THREAD

**Issue by JDWoodcock** [npct/pct-shiny#785]
could we have the data downloads available on the landing page?

**Comment by Robinlovelace** [comment]
Sounds like a good plan to me, many people still don't know they are available.

**Comment by usr110** [comment]
@AnnaGoodman1 could you tell me where the rmd documents for national level datasets for both layers exist? 
For the main page, I think we need to merge both layers in a single document

**Comment by AnnaGoodman1** [comment]
should we maybe just link to national datasets from main page? can't see how linking to all regional ones would easily work, though happy to see ideas

**Comment by AnnaGoodman1** [comment]
rmd docs at: pct-shiny\non_www\tabs\commute and pct-shiny\non_www\tabs\school

**Comment by usr110** [comment]
> should we maybe just link to national datasets from main page

I think so. Regional data would be quite complex.


> rmd docs at: pct-shiny\non_www\tabs\commute and pct-shiny\non_www\tabs\school

Thanks. I don't see any rmd docs there. Have they been moved somewhere else?

See for instance here: https://github.com/npct/pct-shiny/tree/master/non_www/tabs/commute

**Comment by nikolai-b** [comment]
We could improve the regional pop ups on the front page and put regional download links in them.

**Comment by AnnaGoodman1** [comment]
@usr110 , you are right - but those are the download page files I think. I guess html not .rmd...?

**Comment by usr110** [comment]
> 
> 
> @usr110 , you are right - but those are the download page files I think. I guess html not .rmd...?

Thanks. I will work with html then.

**Comment by Robinlovelace** [comment]
Also just picked up on this. Sounds like you're onto a solution @usr110, let us know if you'd like to test/review any changes. Will make the data that bit more accessible, amazing how few people know the data downloads are there.

**Comment by Robinlovelace** [comment]
> What about instead having a link to the regional data download page in the pop-up box, as shown in this image

I'm happy with that it sounds like it could be tricking and time-consuming to implement. My understanding is that @usr110 would work on it and for me it's a question of how long it would take. Previous comments from Ali, e.g. that it 

> will increase complexity

and

> might not work smoothly

led me to proposed a simpler stop-gap solution that points people to the data. I think the file structure is fairly self explanatory and that advertising national and regional data has +s that outweigh potential -s of people being confused when they are directed to GitHub. Are there additional -s of my suggestion, of simply adding a sentence on the main page directing people to the downloads, that we've not considered? 

Also, adding the links to the main page is not incompatible with doing it in the popups. We could do (and I tentatively advocate doing) both. But the former option is simpler from user and developer perspectives.

Would also be keen to here Nikolai's take: would Anna's suggestion in the image above be time consuming to implement @nikolai-b ? All for it if so, just need to know who will do it and an estimate of how long it would take. 

In the meantime I suggest going with the simple message I suggest here https://github.com/npct/pct-shiny/issues/785#issuecomment-530807902 unless there are unforseen disadvantages that I've not considered.

**Comment by AnnaGoodman1** [comment]
ok maybe I misunderstood - I am now not clear what you were suggesting Robin. you looked like you were suggesting linking to specific geojson/Rds files on a pop up, which I can't see working - I think we should link to the pages not to files. And I don't think a good idea to link to github, if that is what you are suggesting - but as I say, not sure.

just having a static sentence/box advertising the existence of these files seems fine to me as a minimal but useful stopgap: 'Did you know you can download regional and national files? Click on a region and then choose the relevant tab along the top'

**Comment by Robinlovelace** [comment]
> Did you know you can download regional and national files? Click on a region and then choose the relevant tab along the top

I think that simple addition to the front page would be great.

**Comment by Robinlovelace** [comment]
Quick check-up on this, are you up for giving this a bash @usr110 after great work on adding the Essential doc? Otherwise I can have a bash. Heads-up @nikolai-b I think we should fix this before merging https://github.com/npct/pct-shiny/pull/836
